.hooks/pre-commit

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+export PATH=~/.local/bin:$PATH
```

Comment:
what does this fix?

Reply:
This is the UNIX location for poetry, and it makes the script to recognize `poetey` in multiple envs.
For example, the commit button in VSCode is missing this path.
I removed the export